pkg/virt-config/config-map.go

```diff
@@ -292,6 +295,18 @@ func setConfig(config *Config, configMap *k8sv1.ConfigMap) error {
 		return fmt.Errorf("invalid value for permitSlirpInterfaces in config: %v", permitSlirp)
 	}
 
+	// disable bridge
+	permitBridge := strings.TrimSpace(configMap.Data[PermitBridgeInterfaceOnPodNetwork])
+	switch permitBridge {
+	case "":
+		// keep the default
+	case "false":
+		config.PermitBridgeInterfaceOnPodNetwork = false
+	case "true":
+		config.PermitBridgeInterfaceOnPodNetwork = true
+	default:
+		return fmt.Errorf("invalid value for permitSlirpInterfaces in config: %v", permitSlirp)
```

Comment:
the error message refers to the wrong name of the attribute; and you are using the wrong variable name.